static/css/legacy.less

```diff
@@ -1416,6 +1416,12 @@ tr {
       padding-left: 30px !important;
     }
   }
+  td.valign_mid{
+    vertical-align:"middle"
+  }
+  td.valign_top{
```

Comment:
I think this is a step in the right direction but I suggest calling this class .column--aligned-top to be consistent with the other BEM naming we've been using recently.

Reply:
DONE